python/tidy.py

```diff
@@ -100,8 +100,10 @@ def check_length(file_name, idx, line):
     if file_name.endswith(".lock") or file_name.endswith(".json"):
         raise StopIteration
     max_length = 120
-    if len(line.rstrip('\n')) > max_length:
-        yield (idx + 1, "Line is longer than %d characters" % max_length)
+    current_line = line.rstrip('\n')
+    if 'use' in current_line and '{' in current_line or '}' in current_line:
```

Comment:
Should this not be...
`not 'use' in current_line or '{' in current_line and '}' in current_line`

And maybe you want startswith 'use'